test/katex-spec.js

```diff
@@ -2566,23 +2581,6 @@ describe("A smash builder", function() {
     });
 });
 
-describe("A document fragment", function() {
-    it("should have paddings applied inside an extensible arrow", function() {
-        const markup = katex.renderToString("\\tiny\\xrightarrow\\textcolor{red}{x}");
-        expect(markup).toContain("x-arrow-pad");
-    });
-
-    it("should have paddings applied inside an enclose", function() {
-        const markup = katex.renderToString(r`\fbox\textcolor{red}{x}`);
-        expect(markup).toContain("boxpad");
-    });
-
-    it("should have paddings applied inside a square root", function() {
-        const markup = katex.renderToString(r`\sqrt\textcolor{red}{x}`);
-        expect(markup).toContain("padding-left");
-    });
-});
```

Comment:
Why were these test cases removed?

Reply:
@kevinbarabash They're test cases added in #1707, which relied on edge-case and previous (wrong) behaviors.